README.md

```diff
@@ -0,0 +1,27 @@
+# social-media-client
```

Comment:
There is more you can add to this readme file. See this template https://gist.github.com/Igormandello/57d57ee9a9f32a5414009cbe191db432

Reply:
README file is updated